wgsl/index.bs

```diff
@@ -17809,40 +17809,98 @@ fn atomicLoad(atomic_ptr: ptr<AS, atomic<T>, read_write>) -> T
 Returns the atomically loaded the value pointed to by `atomic_ptr`.
 It does not [=atomic modification|modify=] the object.
 
-### Atomic Store ### {#atomic-store}
+### `atomicStore` ### {#atomic-store}
 
 ```wgsl
 fn atomicStore(atomic_ptr: ptr<AS, atomic<T>, read_write>, v: T)
 ```
 
 Atomically stores the value `v` in the atomic object pointed to by `atomic_ptr`.
 
-### Atomic Read-modify-write ### {#atomic-rmw}
+### Atomic Read-modify-write Functions ### {#atomic-rmw}
+
+Each function performs the following steps atomically:
+
+1. Load the original value pointed to by `atomic_ptr`.
+2. Obtains a new value by performing the operation (e.g. max) from the function
+    name with the value |v|.
+3. Store the new value using `atomic_ptr`.
+
+Each function returns the original value stored in the atomic object before the operation.
+
+#### `atomicAdd` #### {#atomic-add}
 
 ```wgsl
 fn atomicAdd(atomic_ptr: ptr<AS, atomic<T>, read_write>, v: T) -> T
+```
+
+Atomically performs an addition operation on the atomic object pointed to by `atomic_ptr`
```

Comment:
I've been debating with myself about the value of linking or writing the expanded code for these RMW operations. I could see a few ways to do this:
1. Link to the appropriate expression or builtin function (may need some new anchors in expression tables)
2. Add the example code (e.g. `*atomic_ptr + x`)
3. Have an example polyfill such as:
```
// All operation performed atomically
fn atomicAdd(atomic_ptr: ptr<AS, atomic<T>, read_write>, v : T) -> T {
  let old = *atomic_ptr;
  *atomic_ptr = old + v;
  return old
}
```

Thoughts?

Reply:
@alan-baker certainly, and I think polyfills simply work the best: linking is ofc a solution but even if we add anchors, it is still not clear enough as to returning of old etc., example relies more on comments, however polyfill is quite straightforward to expose to reader a bit more. WDYT of latest commit? TYVM!